src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Linking a SCSS file directly in an HTML document will not work because browsers do not understand SCSS syntax. You need to compile your SCSS to CSS and link the resulting CSS file instead.